README.md

```diff
@@ -1 +1,52 @@
-# portfolio-project1
\ No newline at end of file
+# Portfolio Project
+
+> Set up a Portfolio project.
+
+![image](https://user-images.githubusercontent.com/66781780/124994067-543c9d80-e056-11eb-94f7-b0c6dc6e3965.png)
+
+In this project, we will set up a "A portfolio-project" repository. No complex coding is required for this exercise. Our goal here is to master all of the tools and best practices we learned about in previous steps.
+
+## Built With
+
+- HTML
+- CSS
+
+## Getting Started
+
+To get a local copy up and running follow these simple example steps.
+
+`https://github.com/balqees0325/portfolio-project1.git`
+
+change into the working directory.
+
+`cd porfolio project`
+
+### Prerequisites
+
+- Node.JS
+
+## Author
+
+👤 **Balqees Ibrahim**
+
+- GitHub: [@balqees0325](https://github.com/balqees0325)
+- Twitter: [@qeescode](https://twitter.com/qeescode)
+- LinkedIn: [Balqees Ibrahim](https://www.linkedin.com/in/balqees-ibrahim-911a81204/)
+
+## Live Demo
+
+- [Live link](https://balqees0325.github.io/portfolio-project1/index.html)
+
```

Comment:
- [ ] Optional

* Kindly, fix this issue, your live link is not working
![live](https://user-images.githubusercontent.com/55635977/125100463-6a3f7200-e0d9-11eb-8c1b-1e4c30f25c15.png)

Use this, https://raw.githack.com/balqees0325/portfolio-project1/about/index.html# , but next time pay attention on it.